src/driver/amdxdna/aie2_ctx.c

```diff
@@ -1057,10 +1027,17 @@ int aie2_cmd_submit(struct amdxdna_hwctx *hwctx, struct amdxdna_sched_job *job,
 		return ret;
 	}
 
+	job->chain = dma_fence_chain_alloc();
```

Comment:
can we use a local variable instead of adding chain to job data structure?

Reply:
Good point. Will fix it.